### PR TITLE
Add feedback button and functionality

### PR DESCRIPTION
**Pull request by gitchrisadams**
image

### CODE REVIEW

**Comment by gitchrisadams** [comment]
@ocefpaf Would you be able to take a look at this PR when you get a chance and merge if looks good. Thanks.

File: assets/js/feedback.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      }
+      else
+        if ($(v).attr('id') === 'feedback-email' && !validateEmail($(v).val())) {
+          $(v).parent().addClass('has-error').find('span').css('visibility', 'visible');
+          valid = false;
+        }
+    });
+    if (valid)
+      $.ajax({
+        url   : 'https://oceansMap2.asascience.com/?SiteID=1&Name=' + $('#feedback-name').val() + '&Email=' + $('#feedback-email').val() + '&Comment=' + $('#feedback-comment').val(),
```
Comment (ocefpaf) [inline assets/js/feedback.js]: @gitchrisadams I don't know the first thing about JS but my guess is that the message posted on this domain, right?

For this site @jbosch-noaa originally decided to have all the feedback as GitHub issues (top left banner). I don't really have a preference so I'll leave this one for @jbosch-noaa to decided and merge.

Comment (gitchrisadams) [reply assets/js/feedback.js]: Yes, message posted using that url. It matches how the other feedback buttons work for the ioos.us sites.

**Comment by ocefpaf** [comment]
Thanks @gitchrisadams!

**Pull request merged**